pkg/cache/cache_test.go

```diff
@@ -69,185 +125,180 @@ var _ = Describe("Informer Cache", func() {
 		})
 
 		It("should be able to get objects that haven't been watched previously", func() {
-			By("Creating the cache")
-			reader, err := cache.New(cfg, cache.Options{})
+			By("getting the Kubernetes service")
+			svc := &kcorev1.Service{}
+			svcKey := client.ObjectKey{Namespace: "default", Name: "kubernetes"}
+			Expect(informerCache.Get(context.Background(), svcKey, svc)).NotTo(HaveOccurred())
+
+			By("verifying that the returned service looks reasonable")
+			Expect(svc.Name).To(Equal("kubernetes"))
+			Expect(svc.Namespace).To(Equal("default"))
+		})
+
+		It("should support filtering by labels", func() {
+			By("listing pods with a particular label")
+			// NB: each pod has a "test-label" equal to the pod name
+			out := kcorev1.PodList{}
+			Expect(informerCache.List(context.Background(), client.InNamespace(testNamespaceTwo).
```

Comment:
nit: `Expect(...).To(Succeed())` is the same thing, but reads more naturally in the ginkgo output.

Reply:
Done